src/Types.js

```diff
@@ -53,8 +53,10 @@ class Types {
         });
     }
 
-    constructor(deserialize: (value: any) => any) {
-        this.deserialize = deserialize;
+    constructor(deserializer: (value: any) => any) {
+        this.deserialize = (value: any) =>
+            deserializer(value === null ? this.defaultValue() : value);
+
         this.json = {
             key: '',
             alias: 'Name',
```

Comment:
`defaultValue` should be initialized to `null`

Reply:
`defaultValue(value: any) {
...     return this.get('defaultValue') || null;
    }` is equivalent no (without defining the value itself) ?